### PR TITLE
Read image dimensions

### PR DESCRIPTION
**Pull request by penman**

### CODE REVIEW

File: lib/jekyll-gallery-generator.rb [hunk 1]
```diff
@@ -207,6 +207,10 @@ def initialize(site, base, dir, gallery_name)
       self.data["best_image"] = gallery_config["best_image"] || best_image
       best_image_path = File.join(dir.to_s, best_image.to_s)
       self.data["date_time"] = get_exif_date_time(best_image_path)
+      self.data["image_dimensions"] = @images.map do |image|
+        exif = EXIFR::JPEG.new(File.join(dir, image))
```
Comment (ggreer) [inline lib/jekyll-gallery-generator.rb]: FYI, this can throw if there's no EXIF data (common for screenshots & backgrounds). I'll fix it after merging.

**Comment by ggreer** [comment]
Thanks for both PRs!

**Comment by ggreer** [comment]
OK, I've moved some stuff around to improve performance and make the EXIF data a little more general. See https://github.com/ggreer/jekyll-gallery-generator/commit/84f78ae47292c223ec52e8c644268d986dff29b7 and https://github.com/ggreer/jekyll-gallery-generator/commit/7e24eed2713b908934136119ff0dbd8d3a705856 for the exact changes. If you want to get at the image dimensions in a template, you'll want `image.exif.width` and `image.exif.height`.

**Comment by penman** [comment]
Cool, thanks!

**Pull request merged**